easybuild/easyconfigs/g/gdbgui/gdbgui-0.13.1.2-GCCcore-8.2.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,96 @@
+easyblock = 'PythonBundle'
+
+name = 'gdbgui'
+version = '0.13.1.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://gdbgui.com'
+description = """Browser-based frontend to gdb (gnu debugger). Add breakpoints, view the stack, visualize data
+structures, and more in C, C++, Go, Rust, and Fortran. Run gdbgui from the terminal and a new tab will open in your
+browser."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+builddependencies = [('binutils', '2.31.1')]
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('GDB', '8.3', '-Python-3.7.2'),
+]
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+use_pip = True
+
+exts_list = [
+    ('greenlet', '0.4.15', {
+        'source_urls': ['https://pypi.python.org/packages/source/g/greenlet/'],
+        'checksums': ['9416443e219356e3c31f1f918a91badf2e37acf297e2fa13d24d1cc2380f8fbc'],
+    }),
+    ('pygdbmi', '0.9.0.1', {
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/cs01/pygdbmi/archive/'],
+        'checksums': ['9c3b1757dd11d6d90a2bf47a22277f6c204646402950fa981daffe740c2493ad'],
+    }),
+    ('gevent', '1.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/g/gevent/'],
+        'checksums': ['1eb7fa3b9bd9174dfe9c3b59b7a09b768ecd496debfc4976a9530a3e15c990d1'],
+    }),
+    ('itsdangerous', '1.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/i/itsdangerous/'],
+        'checksums': ['321b033d07f2a4136d3ec762eac9f16a10ccd60f53c0c91af90217ace7ba1f19'],
+    }),
+    ('Flask-Compress', '1.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/flask-compress/'],
+        'checksums': ['468693f4ddd11ac6a41bca4eb5f94b071b763256d54136f77957cfee635badb3'],
+    }),
+    ('Flask-SocketIO', '2.9.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/flask-socketio/'],
+        'checksums': ['f49edfd3a44458fbb9f7a04a57069ffc0c37f000495194f943a25d370436bb69'],
+    }),
+    ('Flask', '0.12.4', {
```

Comment:
@Micket there is already an easyconfig for `Flask`. given that several other codes need it too, maybe its better to update it (and add Flask-Compress, Flask-SocketIO) and add it as dependency for gdbgui?

Reply:
@smoors gdbgui unfortunately has a requirement for `Flask < 1.0` though
https://github.com/cs01/gdbgui/blob/master/setup.py#L12
as well as a requirement on an older Flask-SocketIO